sphinxcontrib/chapeldomain.py

```diff
@@ -64,6 +64,49 @@
 #     """Node for a "returns" annotation."""
 # nodes._add_node_class_names([chapel_desc_returns.__name__])
 
+class ChapelTypedField(TypedField):
+    """Override TypedField in order to change output format."""
+
+    def make_field(self, types, domain, items):
+        """Copy+Paste of TypedField.make_field() from Sphinx version 1.2.3. The first
+        and second nodes.Text() instance are changed in this implementation to
+        be ' : ' and '' respectively (instead of ' )' and '').
```

Comment:
The last '' section in this comment is supposed to be just empty, right?  What is the change between the second instances, then?